### ISSUE TITLE
Blatt 9

### ISSUE THREAD

**Issue by konstantinkobs** [konstantinkobs/Stochastik-II#8]
Hey, also dieses Mal bin ich etwas verwirrt bei diesem Blatt...

Mit Aufgabe 1 habe ich mich noch nicht so ausgiebig beschäftigt, das werde ich noch tun. Bei Aufgabe 2 allerdings weiß ich nicht mal, welches *Münzwurfbeispiel* eigentlich gemeint ist. Anzahl von Köpfen bei n Würfen? Wie rechnet man denn da weiter?

**Comment by step5589** [comment]
Genau, mit "unserem münzwurfbeispiel" war bis jetzt immer die Anzahl von
Kopf bei n Würfen gemeint, zum zweiten Teil der Frage muss ich nochmal ein
bisschen überlegen :)

Am 11. Januar 2015 um 15:55 schrieb Konstantin Kobs <
notifications@github.com>:

> Hey, also dieses Mal bin ich etwas verwirrt bei diesem Blatt...
>
> Mit Aufgabe 1 habe ich mich noch nicht so ausgiebig beschäftigt, das werde
> ich noch tun. Bei Aufgabe 2 allerdings weiß ich nicht mal, welches
> *Münzwurfbeispiel* eigentlich gemeint ist. Anzahl von Köpfen bei n
> Würfen? Wie rechnet man denn da weiter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/konstantinkobs/Stochastik-II/issues/8>.
>



-- 
Stefanie Kruse
Horner Landstr. 159a
22111 Hamburg
Tel: +49 151 6144 4214
